public/index.html

```diff
@@ -10,6 +10,20 @@ <h1>A wild webpage appears...</h1>
     <div id="list-container"></div>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
     <script>
+      // our jquery template string
+      function getList(data) {
+        var compiled = '';
+        data.forEach(item => {
+          compiled += `
+            <div class="row">
+              <div class="col-xs-6"><strong>${item.name}</strong></div>
+              <div class="col-xs-6">${item.description}</div>
+            </div>
```

Comment:
Should we add price?

Reply:
I didn't because this week appeared to be using postman not the browser, but i think i'm fine with it either way